manager/src/main/resources/schemas/actions/catalog.json

```diff
@@ -0,0 +1,6 @@
+[
+  "KafkaTopic.json",
+  "SendToBridge.json",
+  "Slack.json",
+  "Webhook.json"
```

Comment:
Can you add a test case to verify if this file matches the list of the resources?

Reply:
done